src/Exception/SchemaNotReadException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Cycle\Exception;
+
+use RuntimeException;
+use Yiisoft\FriendlyException\FriendlyExceptionInterface;
+
+class SchemaNotReadException extends RuntimeException implements FriendlyExceptionInterface
+{
+    public function __construct()
+    {
+        parent::__construct('Cycle Schema not read.');
```

Comment:
Am I correct about the meaning?

```suggestion
        parent::__construct('Cycle Schema can not be read.');
```